### PR TITLE
feat(components): update icon colors [JOB-44183]

### PR DESCRIPTION
**Pull request by chris-at-jobber**
## Motivations

In the process of standardizing our Button icon colors on mobile, we ran across an issue where the `checkmark` icon, despite being rendered using an `iconColor` of green everywhere it is used, did not have this as a "system" color.

While I came here to update `checkmark` to use the semantic `success` color, I also noticed some long-standing cleanups that should be done.

### Added

- `success` for Checkmark
image

### Changed

- re-ordered a few items to group core workflow objects and external filetype icons better 
- tidied up the docs intro (had a typo, was overly wordy)

### Removed

- No more `brown` for Edit/pencil
image

## Testing

Check the `checkmark` icon and see that it is green, see that edit is not brown

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/design/src/icons/Icon.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .icon {
-  fill: var(--color-greyBlue--dark);
```
Comment (chris-at-jobber) [inline packages/design/src/icons/Icon.css:2]: Color change to match intended designs

File: packages/design/src/icons/Icon.css [hunk 2]
```diff
@@ -67,28 +71,26 @@
...
-.excel {
-  fill: rgb(31, 115, 71);
+.video {
+  fill: var(--color-lightBlue);
 }
 
-.file {
-  fill: var(--color-greyBlue);
+/* non-Atlantis colors to match brand colors of these file-types */
+
```
Comment (MichaelParadis) [inline packages/design/src/icons/Icon.css:91]: Super small nitpick but I think the line between the comment and the CSS class should be removed

File: packages/design/src/icons/Icon.css [hunk 3]
```diff
@@ -67,28 +71,26 @@
...
   fill: var(--color-blue);
 }
 
-.excel {
-  fill: rgb(31, 115, 71);
+.video {
+  fill: var(--color-lightBlue);
 }
 
-.file {
```
Comment (chris-at-jobber) [inline packages/design/src/icons/Icon.css:90]: remove file because it now matches default icon color

**Comment by chris-at-jobber** [comment]
@rodrigoeidelvein try the [Cloudflare build](https://a2ae8b14.atlantis.pages.dev/components/icon) - you might need to run `npm ci` on your local to get the changes to the `design` package 🤔

File: packages/design/src/icons/Icon.css (hunk 2 above)
Comment (chris-at-jobber) [reply packages/design/src/icons/Icon.css:91]: Is that a thing? I thought best practice was to add lines between for readability?

Comment (MichaelParadis) [reply packages/design/src/icons/Icon.css:91]: @chris-at-jobber nevermind. I looked at examples and only found one that followed the way I suggested.

**Review by rodrigoeidelvein — APPROVED** [review]
LGTM!

**Pull request merged**